lib/ts/recipe/thirdparty/providers/utils.ts

```diff
@@ -62,19 +123,19 @@ export async function discoverOIDCEndpoints(config: ProviderConfigForClientType)
     if (config.oidcDiscoveryEndpoint !== undefined) {
         const oidcInfo = await getOIDCDiscoveryInfo(config.oidcDiscoveryEndpoint);
 
-        if (oidcInfo.authorisation_endpoint) {
+        if (oidcInfo.authorisation_endpoint && config.authorizationEndpoint === undefined) {
```

Comment:
```suggestion
        if (oidcInfo.authorisation_endpoint !== undefined && config.authorizationEndpoint === undefined) {
```

Reply:
same applies to below as well. No implicit checks

Reply:
Done